### PR TITLE
Add open invites

### LINKED ISSUE #102: follow-up to privacy modes: open invites
initially part of #96 but while talking with @cblgh about #100 I realized that the invites implementation needs to be overhauled to conform with https://github.com/ssb-ngi-pointer/rooms2/pull/35 first anyhow.

I _suspect_ we don't strictly need _invite creation_ (which is currently tied to members here) but basically have open consumption that makes people internal users no matter what the `inviteCode`.

So in open mode we could have a `Create Invite` button _somewhere_ that just returns a link with a random code.

### PR DESCRIPTION
**Pull request by cryptix**
fixes #102 
updates #199

### CODE REVIEW

File: web/handlers/admin/invites.go [hunk 1]
```diff
@@ -73,8 +73,6 @@ func (h invitesHandler) create(w http.ResponseWriter, req *http.Request) (interf
 	}
 
 	facadeURL := h.urlTo(router.CompleteInviteFacade, "token", token)
-	facadeURL.Host = h.domainName
-	facadeURL.Scheme = "https"
```
Comment (staltz) [inline web/handlers/admin/invites.go:77]: What happened here? :)

File: web/templates/invite/facade.tmpl [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {{ define "title" }}{{ i18n "InviteFacadeTitle" }}{{ end }}
 {{ define "content" }}
 <div class="flex flex-col justify-center items-center self-center max-w-lg">
-  <p id="welcome" class="text-center mt-8 italic">Welcome to {{.RoomTitle}}! {{i18n "InviteFacadeWelcome"}}</p>
+  <p id="welcome" class="text-center mt-8 italic">{{i18nWithData "InviteFacadeWelcome" "RoomTitle" .RoomTitle}}</p>
```
Comment (staltz) [inline web/templates/invite/facade.tmpl:4]: :+1:

File: web/handlers/admin/invites.go (hunk 1 above)
Comment (cryptix) [reply web/handlers/admin/invites.go:77]: `urlTo` was changed a while ago to set this itself. This here went un-noticd in the refactor and is obsolete now.

https://github.com/ssb-ngi-pointer/go-ssb-room/blob/385b98a3a11a4316e1403e7c62ae7ec91a3f78aa/web/utils.go#L99-L104

Comment (staltz) [reply web/handlers/admin/invites.go:77]: Cool cool

**Review by staltz — APPROVED** [review]

**Pull request merged**